<launch>

	<node pkg="rt_audio_ros" type="rt_audio" name="rt_audio" output="screen"> 
		<!--launch-prefix="xterm -e gdb -\-args"-->
		<param name="save_raw_file" value="false"/>
        <param name="nb_microphones" value="1"/>
	</node>

</launch>
